apps/web/src/components/Member/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link';
+
+import { StyledMember, Card, ProfileImage, H1 } from './styles';
+
+interface MemberProps {
+  name: string;
+  username: string;
+}
+
+export const Member: React.FC<MemberProps> = ({ name, username }) => {
+  return (
+    <StyledMember>
+      <Card>
+        <H1>{name}</H1>
```

Comment:
could we have a more meaninful name for this? like `<Title />` or something similar